bin/radical-pilot-control

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+__copyright__ = "Copyright 2024, http://radical.rutgers.edu"
+__license__   = "MIT"
+
+
+import os
+import sys
+import time
+
+
+# ------------------------------------------------------------------------------
+#
+def run_agent_command(sid, cmd, args):
+
+    import json
+    import socket
+
+    portfile = f'{sbox}/agent_0.ports.json'
```

Comment:
`sbox` is not provided as an argument,
btw, seems that `sid` is not needed here (we use `sbox` instead)

Reply:
Thanks, fixed and added some checks.